environment.yml

```
@@ -119,7 +119,7 @@ dependencies:
 
   # Packages specific to cryo work
   - h5coro~=0.0.6
-  - icepyx==0.7.0
+  - icepyx==0.8.0
```

Minor suggestion to use `~=` (compatible release spec) per https://github.com/CryoInTheCloud/hub-image/issues/46#issuecomment-1542864196.

```suggestion
  - icepyx~=0.8.0
```